File: Codevitas/shivam/codevita/sec.cpp
#include <stdio.h>
#include <string.h>
#include <iostream>
using namespace std;
int main()
{
    int l,i=0,count=0;
    string s;
    cin >> l >> s;
    while (i < l){
        if(s[i] == 'R'){
            if (i != l-1){
                count++;i++;
            }
            else {
                break;
            }
        }
        else {
            while(i < l && s[i] == 'G') i++;
            if (i == l)break;
            else count++;
        }
    }
    cout << count;
}

File: Codevitas/shivam/codevita/sec.py
l =input().split()
s = l[1]
l = int(l[0])
i = 0
count = 0
while (i < l):
	if i == l-1:
		break
	if s[i] == 'R' and i != l-1:
		count += 1
		i+=1
	elif s[i] == 'G':
		while (i < l and s[i] == 'G'):
			i+=1
		if i == l:
			break
		else :
			count += 1
print(count)
